### DISCUSSION TITLE
Timeout

### DISCUSSION THREAD

**Discussion by sdavisNL** [Taxel/PlexTraktSync#1539] [Support]
I have the config file set with this timeout setting:

```yml
plex:
  timeout: 500
```
However everytime I run the app it gives me an error in the log that reads:

Read timed out. (read timeout=30)

It doesn't seem like the setting in the config file is doing anything, it's going back to the default of 30.

Version: 0.26.8

**Comment by glensc** [comment]
Bugs must be reported as issues. I belive this is already fixed
